bjoern/request.c

```diff
@@ -23,6 +28,7 @@ Request* Request_new(ServerInfo* server_info, int client_fd, const char* client_
   request->client_addr = _Unicode_FromString(client_addr);
   http_parser_init((http_parser*)&request->parser, HTTP_REQUEST);
   request->parser.parser.data = request;
+  request->parser.field = NULL;
```

Comment:
I don't think we actually have to NULL the field here?

It's done in `on_message_begin`